fluentd-gcp-image/Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 gem 'fluentd', '~>0.12.32'
 gem 'fluent-plugin-record-reformer', '~>0.8.3'
 gem 'fluent-plugin-systemd', '~>0.0.8'
-gem 'fluent-plugin-google-cloud', '~>0.6.10'
+gem 'fluent-plugin-google-cloud', '~>0.6.11'
```

Comment:
Note: [v0.6.11](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/releases/tag/v0.6.11) included https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/179, which broke private key authentication. It was reverted in https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/198, which is included in [v0.6.12](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/releases/tag/v0.6.12). If you don't care about private key authentication, v0.6.11 is fine, but just thought I'd let you know.

Reply:
Thanks for letting me know. I'll use 0.6.12 then.